### DISCUSSION TITLE
Mysql updating only one table deletes all others

### DISCUSSION THREAD

**Discussion by Townieeee** [Budibase/budibase#13662] [Help]
I saw this post -Mysql updating only one table deletes all others #11182

Summary: You want to fetch 1 table then suddenly your other tables are "gone".

It talks about BB HIDING the other tables. How does one "UNHIDE" those again after you have fetched the 1 you want? 
Is this some setting? I have navigated around and cannot get my "other" tables back. 
Any help appreciated. Thx

**Comment by ConorWebb96** [comment]
Hey @Townieeee,

The fetching area will re-fetch everything that is selected. Even if you add a new table you need to ensure that all the other tables are still checked whenever fetching a new one. Otherwise like you said it will remove the other tables from view.

If you wish to see them again, simply fetch your tables again and ensure that all the tables you wish to have visible are checked.

Hopefully this helps.

**Comment by Townieeee** [comment]
Thx . So it does not mess with any of the tables that may have had something done to them? - like a relationship made with another table
